CMakeLists.txt

```diff
@@ -298,6 +298,8 @@ add_library(${TARGET_NAME} SHARED
     $<TARGET_OBJECTS:cl_headers>
 )
 
+set_msvc_crt_flags(${TARGET_NAME})
```

Comment:
can we check if macro set_msvc_crt_flags exists, rather than adding the implementation of the macro?

Reply:
good idea! I will update it.

Reply:
or add a new cmake option to enable this macro call. And this cmake option should be OFF by default.